examples/charm++/load_balancing/stencil3d/stencil3d.C

```diff
@@ -324,7 +324,7 @@ class Stencil: public CBase_Stencil {
         work = 10.0;
 
 #ifndef _MSC_VER
```

Comment:
I would be in favor of just removing these, rather than commenting them out

Reply:
Done